app-crypt/gnupg/gnupg-2.4.7.ebuild

```diff
@@ -36,7 +36,6 @@ DEPEND="
 	>=dev-libs/libgpg-error-1.46
 	>=dev-libs/libksba-1.6.3
 	>=dev-libs/npth-1.2
-	>=net-misc/curl-7.10
```

Comment:
Thanks. In future, please make sure to update at least newer versions. It also needs a revbump (which I've done) and some analysis which I added in the bug.

Reply:
> Thanks. In future, please make sure to update at least newer versions. It also needs a revbump (which I've done) and some analysis which I added in the bug.

was about to push the revbumps, yes.

you were much faster :+1:

Reply:
Thanks for the spot! It was a nice one.